compiler/src/dotty/tools/dotc/transform/init/Cache.scala

```diff
@@ -75,6 +75,12 @@ class Cache[Config, Res]:
    */
   protected var changed: Boolean = false
 
+  /** Whether any value in the cache was accessed after being added.
+  *   If no cached values are used after they are added for the first time
+  *   then another iteration of analysis is not needed.
+  */
```

Comment:
Consider make it more precise in the comment: Whether any value in the **output** cache (`this.current`).

Minor: the comment `*` is not aligned.